### PR TITLE
Feat(items/currency): Block direct transfer by site settings

### PR DESCRIPTION
**Pull request by SpeedyD**
Don't want your users to directly transfer items or currency, and instead pass it through a trade?

Aight, just change your site settings. :)

### CODE REVIEW

File: app/Console/Commands/AddSiteSettings.php [hunk 1]
```diff
@@ -82,6 +82,10 @@ public function handle() {
         $this->addSiteSetting('deactivated_key', 0, 'Optional key to view the deactivated list. Enter "0" to not require one.');
 
         $this->addSiteSetting('comment_dislikes_enabled', 0, '0: Dislikes disabled, 1: Dislikes enabled.');
+        
+        $this->addSiteSetting('can_transfer_currency_directly', 1, 'Can users directly transfer currency to other users without trading? 0: No, they cannot, 1: Yes, they can.');
```
Comment (itinerare) [inline app/Console/Commands/AddSiteSettings.php:86]: These descriptions should probably be written more dryly/in the fashion of the existing site settings text, for consistency.

Comment (SpeedyD) [reply app/Console/Commands/AddSiteSettings.php:86]: I based this style of writing approximately off of how the blacklist / deactivation options are written..

May I request a suggestion for how to rewrite this?

Comment (itinerare) [reply app/Console/Commands/AddSiteSettings.php:86]: Hmm, maybe something like... `Whether or not users can directly transfer [items/currency] to other users without trading. 0: Users cannot directly transfer [items/currency]. 1: Direct [item/currency] transfers are allowed.`

Comment (SpeedyD) [reply app/Console/Commands/AddSiteSettings.php:86]: I will copy paste this in. 👍

**Review by itinerare — APPROVED** [review]

**Review by AW0005 — APPROVED** [review]

**Comment by itinerare** [comment]
Just needs conflict resolution.

**Comment by SpeedyD** [comment]
> Just needs conflict resolution.

Done and done.

**Pull request merged**